homeassistant/components/nuki/__init__.py

```diff
@@ -45,6 +46,8 @@ def _update_devices(devices):
                 device.update(level)
             except RequestException:
                 continue
+            except NukiUpdateException:
+                continue
```

Comment:
Do we want to log something here with low severity, or are these errors expected and OK to hide completely?

Reply:
I think it should be logged as warning

Reply:
What do you think the message should be?
Or just logging the exception as it is

Reply:
There is the style guidelines from Home-Assistant: https://developers.home-assistant.io/docs/development_guidelines?_highlight=logging#log-messages

I'd suggest something like
`_LOGGER.warn("Failed to update nuki device: %s", device.unique_id")`

Otherwise you can just log the exception with it's message, since [pynuki prints a similar message](https://github.com/pschmitt/pynuki/blob/22433b35ecc15422e756cc06c2489c9e4e8ab5c3/pynuki/device.py#L82).